packages/@okta/vuepress-site/docs/guides/protect-your-api/main/aspnetcore3/samplecode.md

```diff
@@ -1,2 +1 @@
-* [ASP.NET Core Web API quickstart](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-3.1&tabs=visual-studio) to build a Web API project
-* [ASP.NET Core example repo](https://github.com/okta/samples-aspnetcore/tree/master/samples-aspnetcore-3x/resource-server) for a finished sample app that you can download
+[Finished sample project](https://github.com/okta/samples-aspnetcore/tree/master/samples-aspnetcore-3x/resource-server).
```

Comment:
```suggestion
[Finished sample project](https://github.com/okta/samples-aspnetcore/tree/master/samples-aspnetcore-3x/resource-server)
```

The ending period renders a little funny:
<img width="241" alt="image" src="https://user-images.githubusercontent.com/99954/155759852-5d531bbb-860e-45f8-b22e-3fd08a433aef.png">

The other versions of this guide do not have a period

Reply:
I've gone through and removed all such weird dots at the ends of these links.